capstone/src/test/javascript/search/autocompleteSpec.js

```diff
@@ -1,15 +1,16 @@
-describe('Autocomplete Suggests correct array', () => {
-  it('Autocomplete selects correct item', () => {
-    const fullData = ['Ahoy', 'Hello', 'Wattup'];
-    const relevantData = ['Hello'];
-    const retData = getRelevantDataSuggestions(fullData, 'H');
-    expect(retData).toEqual(relevantData);
+describe('Autocomplete provides correct list element', () => {
+  it('Tests Proper List Element Creation Simple', () => {
+    const arrElt = 'Hello';
+    const val = 'He';
+    const listElmtActual = createListElmt(arrElt, val);
+    const listElmtExpected = '<strong>He</strong>llo';
+    expect(listElmtExpected).toEqual(listElmtActual.innerHTML);
   });
-
-  it('Autocomplete selects multiple correct items', () => {
-    const fullData = ['Ahoy', 'Hello', 'Hi', 'Wattup'];
-    const relevantData = ['Hello', 'Hi'];
-    const retData = getRelevantDataSuggestions(fullData, 'H');
-    expect(retData).toEqual(relevantData);
+  it('Tests Proper List Element Creation Complex', () => {
```

Comment:
How about adding a test for a corner case where the completion is at the end?

Reply:
added!